File: management/src/main/java/io/hexaforce/management/controller/ExamScheduleGanttChartController.java
package io.hexaforce.management.controller;

import java.util.Arrays;
import java.util.List;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.http.HttpStatus;
import org.springframework.http.ResponseEntity;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.ResponseBody;
import org.springframework.web.bind.annotation.RestController;

import io.hexaforce.dievas.commons.DievasServerMesseage;
import io.hexaforce.dievas.service.ExamScheduleGanttChartService;
import lombok.extern.slf4j.Slf4j;

/**
 * ExamScheduleGanttChart router function.
 * 
 * @version 1.0.0.BUILD-SNAPSHOT
 * @author T.Tantaka 
 */

@Slf4j
@RestController
@RequestMapping("/api")
public class ExamScheduleGanttChartController {
	
	@Autowired
	private ExamScheduleGanttChartService examScheduleGanttChartService;
	
	@ResponseBody
	@GetMapping(value = "/exam-schedule-gantt-chart")
	public ResponseEntity<List<DievasServerMesseage>> init() {
		log.info("Controller(java): init()");
		String x = examScheduleGanttChartService.init();
		return new ResponseEntity<List<DievasServerMesseage>>(Arrays.asList(new DievasServerMesseage(x)),HttpStatus.OK);
	}
	
}

File: webapp/src/main/resources/static/controller/ExamScheduleGanttChartController.js
'use strict';
/*******************************************************************************
 * ExamScheduleGanttChartController
 ******************************************************************************/
app.controller('ExamScheduleGanttChartController', ['ExamScheduleGanttChartService', '$scope', function (ExamScheduleGanttChartService, $scope) {
  
  $scope.init = function () {
    console.log('ExamScheduleGanttChartController.js: init()');
    ExamScheduleGanttChartService.init();
    $scope.serviceName = ExamScheduleGanttChartService.getInitResult()[0].message;
  }
  
}]);